index.js

```diff
@@ -0,0 +1,57 @@
+const fse = require('fs-extra');
+const path = require('path');
+
+const run = config => {
+
+    if (config.directories && Array.isArray(config.directories)) {
+        const promises = config.directories.map(directoryConfig => copyDirectory(directoryConfig));
+
+        return Promise.all(promises)
+            .then(() => {
+                const message = `${config.directories.length} ${config.directories.length === 1 ? 'directory' : 'directories'} processed`;
```

Comment:
I'd move that ternary out of the template literal. Took me a second to parse what was happening.